config/default.yml

```diff
@@ -538,6 +538,7 @@ Style/HashSyntax:
     - ruby19
     - ruby19_no_mixed_keys
     - hash_rockets
+    - no_mixed_keys
```

Comment:
@rrosenblum added a bunch of documentation here in default.yml and the cop (including some examples).